code/stable/nopcm/SRS/HTML/NoPCM_SRS.html

```diff
@@ -544,10 +544,10 @@ <h2>Abbreviations and Acronyms</h2>
       <div class="section">
         <h1>Introduction</h1>
         <p class="paragraph">
-          Due to increasing costs, diminishing availability, and negative environmental impact of fossil fuels, the demand is high for renewable energy sources and energy storage technology. Solar water heating systems provide a novel way of storing energy.
+          Due to increasing costs, diminishing availability, and negative environmental impact of fossil fuels, the demand is high for renewable energy sources and energy storage technology. Solar water heating systems with no phase change material provide a novel way of storing energy.
         </p>
         <p class="paragraph">
-          The following section provides an overview of the Software Requirements Specification (SRS) for solar water heating systems. The developed program will be referred to as Solar Water Heating System (NoPCM). This section explains the purpose of this document, the scope of the requirements, the characteristics of the intended reader, and the organization of the document.
+          The following section provides an overview of the Software Requirements Specification (SRS) for solar water heating systems with no phase change material. The developed program will be referred to as Solar Water Heating System With No Phase Change Material (NoPCM). This section explains the purpose of this document, the scope of the requirements, the characteristics of the intended reader, and the organization of the document.
```

Comment:
I think @smiths usually likes only the letters used in an acronym to be capitalized. Do you think this is OK @smiths or would you prefer "solar water heating system with No Phase Change Material" (or even "...NO Phase Change Material" since the "o" is used, albeit lowercase)?

In either case, "with" shouldn't be capitalized. I believe we have a function for titleizing phrases while ignoring the smaller words that shouldn't be capitalized; can that be used here?

Reply:
A generative approach really does make you think about every little decision!  That's a good thing, even if it is challenging.  :smile:  @samm82 is correct, the capitalization in the expanded acronym should tell the reader the acronym.  The acronym should have each of the capital letters.  

The name NoPCM is really for us to easily distinguish SWHS from NoPCM.  As a standalone example, it probably should be SWHS_NoPCM.  The NoPCM is like an option for the family of SWHS programs.  If it is easy to do, without too many unintended consequences, I think renaming NoPCM to SWHS_NoPCM would be nice.  That would mean for the specific capitalization question above, we just need to turn "With" into "with".

Reply:
We can wait for Dr. Smith's input but in the mean time I have added a temporary fix but it does not look right. I created an issue regarding how we can properly fix it, see #3535

Reply:
I'm not sure if #3535 really fixes this issue. It seems like we need a way to specify which letters are used to make the abbreviations of concepts, then capitalize, bold, etc. these letters when introducing the acronym (whether in the text or in the table). Capitalizing a concept (which would include capitalizing "with"; see the code snippet below) is different than specifying where its abbreviation comes from.

https://github.com/JacquesCarette/Drasil/blob/1acb8ba1553d16d1c949e3b55049fb2242dfe555/code/drasil-lang/lib/Language/Drasil/NounPhrase.hs#L271-L274

Reply:
I was referring to the capitalization in the case of #3535, you mentioned that the "with" should be lower case. As for the abbreviation, although we may create abbreviation in the printers in some cases this does not apply to the common ideas, we have just been pre-defining the abbreviation when that idea is created.

I will change the name to SWHS_NoPCM if it is ok to move on with changing the example names.

Reply:
I think we might have to relax automation with turning a phrase into an abbreviation.  Not all abbreviations are ALLCAPS.

Reply:
Using `SWHS_NoPCM `as the name leads to issues in the .yaml files where an '_' (underscore) implies a dependency. We could get around this by using a name like `SWHSNoPCM `(a '-' leads to other issues within the haskell files). Let me know if this is ok or should it be changed to something else entirely?

Reply:
I don't love the SWHSNoPCM name, but I'm fine with living with it for now in the interest of resolving this issue.  In the future it would help if we had options for the name.  The default option would be the name we prefer.  This name could have underscores, or dashes, accents, or even special symbols.  The other option would be the "conservative" name that isn't going to cause problems when it is used for a folder name, or in a yaml file etc.